src/core/async.ts

```diff
@@ -48,42 +48,56 @@ export function asyncAction(asyncAction: any): any {
     return createAsyncActionInvoker(asyncAction.name, asyncAction)
```

Comment:
Uhm, can we add an optional first parameter for the name? Not sure if most generator polyfills will retain the function name.